infrastructure/variables.tf

```diff
@@ -1,4 +1,14 @@
 variable "prefix" {
   description = "The prefix used for all resources in this environment"
   default = "test"
+}
+
+variable "loggly_token" {
+  description = "Loggly token"
+  sensitive = true
+}
+
+variable "log_level" {
+  description = "Logging level"
+  default = "ERROR"
```

Comment:
Always differs by team, but personally I'd lean towards defaulting to at least WARNING - that feels like something you'd generally want to hear about if it keeps happening